.travis.yml

```diff
@@ -5,7 +5,12 @@ cache:
   - yarn
 
 python:
-  - "2.7.11"
+  - 2.7.11
+  - 3.5
```

Comment:
3.6 should be good too, no?

Reply:
Sure, but since we don't actually work on Python 3 yet, I think it's a waste of resources to add 3.6 at the moment. I'll add it once we actually pass under Python 3.